content/docs/get-started/install/migrating-3.0.md

```diff
@@ -0,0 +1,164 @@
+
+* `pulumi cancel`
+* `pulumi config refresh`
+* `pulumi destroy`
+* `pulumi import`
+* `pulumi logs`
+* `pulumi preview`
+* `pulumi refresh`
+* `pulumi stack *`
+* `pulumi state`
+* `pulumi up`
+* `pulumi watch`
+
+## Update Your Pulumi Programs
+
+{{< chooser language "javascript,typescript,python,go,csharp" >}}
+
+{{% choosable language "javascript,typescript" %}}
+
+### Update Dependencies
+
+```bash
+npm install @pulumi/pulumi@^3.0.0
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+### Update Dependencies
+
+Modify your `requirements.txt` file to update the Pulumi SDK and related providers as below:
+
+```
+pulumi>=3.0.0,<4.0.0
+pulumi-aws>=4.0.0,<5.0.0
+```
+
+Then run `pip install`:
+
+```bash
+pip install -r requirements.txt
+```
+
+{{% /choosable %}}
+{{% choosable language go %}}
+
+### Update Dependencies
+
+In `go.mod`, you can depend on the Pulumi SDK and related providers as below:
+
+```
+require (
+    github.com/pulumi/pulumi/sdk/v3 v3.0.0
+    github.com/pulumi/pulumi-aws/sdk/v4/go/aws v3.0.0
+)
+```
+
+Then run `go mod download`
+
+{{% /choosable %}}
+{{% choosable language csharp %}}
+
+### Update Dependencies
+
+Update your package reference to the latest version of the SDK:
+
+```csharp
+<PackageReference Include="Pulumi" Version="3.*" />
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+## Automation API
+
+The Pulumi 3.0 launch, moves the "Automation API" out of preview to be GA! This means, we have standardized the namespace requirements for the "Automation API". You can update your
+program to use the following namespaces
+
+{{< chooser language "javascript,typescript,python,go,csharp" >}}
+
+{{% choosable language "javascript,typescript" %}}
+
+```javascript
+// before
+import as automation from "@pulumi/pulumi/automation";
+
+//after
+import as automation from "@pulumi/pulumi/x/automation";
+```
+
+{{% /choosable %}}
+
+{{% choosable language "python" %}}
+
+```python
+#before
+from pulumi.x import automation as auto
+
+#after
+from pulumi import automation as auto
+```
+
+{{% /choosable %}}
+
+{{% choosable language "go" %}}
+
+```go
+//before
+import (
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v2/go/x/auto"
+	"github.com/pulumi/pulumi/sdk/v2/go/x/auto/optdestroy"
+	"github.com/pulumi/pulumi/sdk/v2/go/x/auto/optup"
+)
+
+//after
+import (
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+	"github.com/pulumi/pulumi/sdk/v3/go/auto"
+	"github.com/pulumi/pulumi/sdk/v3/go/auto/optdestroy"
+	"github.com/pulumi/pulumi/sdk/v3/go/auto/optup"
+)
+```
+
+{{% /choosable %}}
+
+{{% choosable language "csharp" %}}
+
+```csharp
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Pulumi.Automation;
+```
```

Comment:
Was there a before here or did C# never use the old namespace?

Reply:
The DotNet stuff is only new and didn't follow the old X namespace work